client/app/locales/en.json

```diff
@@ -92,7 +92,7 @@
   "error": "Error",
   "error proper email": "Given email is not correct",
   "error email empty": "Given email is empty",
-  "Chose the language you want I use to speak with you:": "Choose the language you want to see:",
+  "Choose the language you want I use to speak with you:": "Choose the language you want to see:",
```

Comment:
Please follow the guidelines when it's about changing the localization key. That one is wrong.